pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -247,25 +257,54 @@ func buildUpstreamName(address string) string {
 
 func buildTransport(mesh *meshconfig.MeshConfig, node *model.Proxy) *mccpb.TransportConfig {
 	// default to mesh
-	networkFailPolicy := mccpb.FAIL_CLOSE
+	policy := mccpb.FAIL_CLOSE
 	if mesh.PolicyCheckFailOpen {
-		networkFailPolicy = mccpb.FAIL_OPEN
+		policy = mccpb.FAIL_OPEN
 	}
 
 	// apply proxy-level overrides
-	if policy, ok := node.Metadata[policyCheckAnnotation]; ok {
-		switch policy {
+	if annotation, ok := node.Metadata[policyCheckAnnotation]; ok {
+		switch annotation {
 		case policyCheckEnable:
-			networkFailPolicy = mccpb.FAIL_CLOSE
+			policy = mccpb.FAIL_CLOSE
 		case policyCheckEnableAllow, policyCheckDisable:
-			networkFailPolicy = mccpb.FAIL_OPEN
+			policy = mccpb.FAIL_OPEN
+		}
+	}
+
+	networkFailPolicy := &mccpb.NetworkFailPolicy{Policy: policy}
+
+	if annotation, ok := node.Metadata[policyCheckRetriesAnnotation]; ok {
+		retries, err := strconv.Atoi(annotation)
+		if err != nil {
+			log.Warnf("unable to parse retry limit %v.", annotation)
```

Comment:
cc @costinm : is there a log dampener in pilot? this might be logging on every request

Reply:
Right - and nobody is really watching the logs.

How about a prometheus variable - tracking invalid annotations of each kind for example - which can be at least acted on.